openquake/hazardlib/gsim/bindi_2014.py

```diff
@@ -114,6 +114,34 @@ def _get_style_of_faulting_term(C, ctx):
     return C["sofN"] * NS + C["sofR"] * RS + C["sofS"] * SS
 
 
+def _get_dist_type(GMPE):
+    """
+    Get distance type required for the corresponding class of 
+    Bindi et al. (2014)
+    """
+    # If gsim is [BindiEtAl2014RjbArmenia] strip to [BindiEtAl2014Rjb]
+    toml_str = GMPE._toml.split('\n')[0]
+    gmpe_str = str(GMPE).split('\n')[0].replace('Armenia','')
+    
+    rjb_variants = [str(BindiEtAl2014Rjb()),
+                    str(BindiEtAl2014RjbEC8()), 
+                    str(BindiEtAl2014RjbEC8NoSOF())]
+    
+    for model in rjb_variants:
+        if model == toml_str or model == gmpe_str:
+            dist_type = {'rjb'}
+    
+    rhypo_variants = [str(BindiEtAl2014Rhyp()),
+                      str(BindiEtAl2014RhypEC8()),
+                      str(BindiEtAl2014RhypEC8NoSOF())]
```

Comment:
Using `str` looks like a code smell. Do you want the name of the class? You can use `__name__` in that case.